pkg/util/util.go

```diff
@@ -252,21 +252,21 @@ func GetAndExtractZip(zipURL string, destination string, pathToUnzip string, sta
 			},
 			Filepath: pathToZip,
 		}
-		err := dfutil.DownloadFile(params)
+		err = dfutil.DownloadFile(params)
 		if err != nil {
 			return err
 		}
 
 		defer func() {
-			if err := dfutil.DeletePath(pathToZip); err != nil {
+			if err = fsys.Remove(pathToZip); err != nil && !os.IsNotExist(err) {
```

Comment:
`os.IsNotExist(err)` seems to not be recommended: https://pkg.go.dev/os#IsNotExist

```suggestion
			if err = fsys.Remove(pathToZip); err != nil && !errors.Is(err, fs.ErrNotExist) {
```